src/components/scenes/RequestScene.js

```diff
@@ -249,7 +250,9 @@ export class RequestComponent extends React.PureComponent<Props, State> {
   }
 
   flipInputRef = (ref: RefObject) => {
-    this.flipInput = ref && ref.flipInput ? ref.flipInput.current : null
+    if (ref?.flipInput) {
```

Comment:
ref is the `ExchangedFlipInput` while ref.flipInput is the `FlipInput` component. Current is not there anymore because of how the withTheme() breaks the normal way of passing refs and how i did not use createRef() on it

Reply:
`this.flipInput = ref.flipInput` this crashed the app when you navigate out of the scene. Will still check ref before unmounting for some reason. Interesting interaction to be honest.

Reply:
Why does it crash? It sounds like this component has another problem somewhere else that also needs fixing, since it should be 100% for `this.flipInput` to be null - otherwise it could crash just as easily *before* it mounts as after.

Reply:
Does `this.flipInput = ref?.flipInput` work? I understand now that `ref` itself can be null.